build-faq.md

```diff
@@ -4,4 +4,58 @@ When building on macOS, the build system will make a native Cocoa app by default
 
 # Attempting to build the SDL frontend on macOS fails on linking
 
-SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
\ No newline at end of file
+SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
+
+# Windows build process
+
+For the various tools and libraries, follow the below guide to ensure easy, proper configuration for the build environment:
+
+#### clang
+
+This may be installed via a Visual Studio installer packages instead of built from source.
```

Comment:
Clang can be downloaded from the official LLVM website(https://releases.llvm.org/download.html; "Pre-Built Binaries", Windows 32/64-bit). An official release is better than one provided by a third party.

Reply:
Seems reasonable to me. I thought it was between building from source or using VS. I agree with you.